main.js

```diff
@@ -24,7 +24,11 @@ column1.addEventListener('click', addGarment);
 outFitInput.addEventListener('input', disableSaveButton);
 column1.addEventListener('click', placeBackground);
 saveOutfitBtn.addEventListener('click', saveOutfit);
-column3.addEventListener('click', removeSavedCard);
+column3.addEventListener('click', function() {
+  removeSavedCard(event)
+  removeCardFromLocalStorage(event);
+});
+
```

Comment:
Here I created an anonymous function in order to pass multiple functions to this event listener.